plugins/kosync.koplugin/main.lua

```diff
@@ -393,6 +406,11 @@ function KOSync:setCustomServer(server)
     self.settings.custom_server = server ~= "" and server or nil
 end
 
+function KOSync:setHostname(hostname)
+    logger.dbg("KOSync: Setting custom hostname to:", hostname)
+    self.settings.kosync_hostname = hostname ~= "" and hostname or nil
```

Comment:
To delete the custom hostname you need to allow blank input:
https://github.com/koreader/koreader/blob/9017bccabaa0fbc5ac8c4199b54c4d3d5f6b2a25/frontend/ui/widget/container/inputcontainer.lua#L381-L386

Reply:
@rspilk Could you please resolve this? If I understand correctly it only needs to be made simpler like this?

```suggestion
    self.settings.kosync_hostname = hostname
```

Reply:
This this cause hostname to be reevaluated as Device.model though? I thought it had to be nil in order for that to work and leaving it empty would be ""?

I looked at adding allow_blank_input = true into the tap input function but I couldn't seem to make that work in my testing. I will try a bit on it this weekend when I have a bit of time to put into getting my environment set up again